research/conditional/utils/conditional_trainer.py

```diff
@@ -84,6 +89,8 @@ class ConditionalTrainer:
     rank: Optional[int] = None
     start_step: int = 0
     checkpoint: Optional[dict[str, torch.Tensor]] = None
+    scheduler_trapezoidal_slides: dict = (None,)
+    args_overload: Optional[dict] = None
```

Comment:
`args_overload` -> `args_override`. Everywhere

Reply:
overwrite, I'm not sure about this one, but i will go with your intuition

Reply:
done

Reply:
just for the records, overwrite would be great as well :)